content/en/docs/KRE/user/50_krt_v2/_index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: "KRT V2"
+linkTitle: "KRT V2"
+weight: 50
+description: >
+  Understading how the new version V2 of KRT works
+---
+
+- [Branched workflows](#branched-workflows)
+- [Reactive approach](#reactive-approach)
+- [Exitpoint](#exitpoint)
+- [KRT YAML File](#krt-yaml-file)
+- [Diagram of an architecture defined by KRT V2](#diagram-of-an-architecture-defined-by-krt-v2)
+
+## Branched workflows
+
+The main purpose of KRT V2 is to bring branched workflows on board. This means nodes are now capable
+of subscribing to several other nodes at the same time, thus workflows are now capable of branching out.
+
+It is advised to previously read and fully understand [KRT](40_krt) before moving on to V2, as this page focuses on explaining V2 related changes and new concepts.
+
+## Diagram of an architecture defined by KRT V2
+
+Here is an example of an architecture defined by a KRT V2 file, this example is taken from our KRT V2 demo repo:
+
+{{< imgproc diagram_krt_v2 Resize "1200x" />}}
+
+## Reactive approach
+
+Nodes will publish their results onto their own subject, is up to other nodes whether to subscribe and process its messages or not. This allows for organic growth as nodes can be developed then hooked up to other nodes freely.
+
+### Multiple responses
+
+Nodes now also have the ability to publish several to none messages from the same request. So now the _handler_ contract should look like this:
+
+- Golang: `handler(ctx *kre.HandlerContext, data *any.Any) error`
+
+- Python: `async def default_handler(ctx, data):`
+
+Messages will be published using the function _SendOutput_ provided by the context.
+
+### Filter messages
+
+Nodes can choose to which subtopic from their own subject they publish to. At the same time, they can choose to take messages from any subtopic of another node. This allows users to filter out messages and reduce stress on nodes.
+
+Nodes can subscribe to one or more subtopics from another node and to various nodes at the same time. To help with filtering, users can set up which _handlers_ are to be executed depending on the request's origin. Take into consideration nodes can only discern the node from which the request proceeds, not the subtopic or topic. For setup guidance consult [Runner SDK]({{< relref "docs/KRE/user/60_kais_runner_sdk" >}}).
+
+## Exitpoint
+
+The exitpoint is a user defined node and can take up any name, however it has to be later assigned in the krt.yaml file to the exitpoint attribute. Users can only define one exitpoint as this is the final node to which the entrypoint will subscribe.
+
+An exitpoint is required in all workflows. As the exitpoint is the only node that will communicate back to the entrypoint, it is recommended to handle merge strategies in it. For example, if users wish to implement "early exit" and "early reply" strategies, they should subscribe the exitpoint to nodes emitting these types of messages.
```

Comment:
Early exits and early replies are not an example of merge strategies, are other use cases of the exitpoint.

Reply:
I didn't write it with that intention, but now I see the "for example" as misleading, I'll change it to a more separate manner.